### ISSUE TITLE
Unable to create groups with fbcreate

### ISSUE THREAD

**Issue by KamalGalrani** [bitlbee/bitlbee-facebook#120]
I always get a `Error: (#2) Service temporarily unavailable` while trying to create groups with fbcreate.

Here is the &bitlbee channel output

```
15:16 -!- singularity [singularity@localhost] has joined &bitlbee
15:16 -!- Topic for &bitlbee: Welcome to the control channel. Type help for help information.
15:16 -!- Topic set by root [root@localhost] [Fri Jan 20 15:16:26 2017]
15:16 [Users &bitlbee]
15:16 [@root] [@singularity] 
15:16 -!- Irssi: &bitlbee: Total of 2 nicks [2 ops, 0 halfops, 0 voices, 0 normal]
15:16 <@root> Welcome to the BitlBee gateway!
15:16 <@root>  
15:16 <@root> Running BitlBee 3.4.2-0ubuntu1
15:16 <@root>  
15:16 <@root> If you've never used BitlBee before, please do read the help information using the help command. Lots of FAQs are answered there.
15:16 <@root> If you already have an account on this server, just use the identify command to identify yourself.
15:16 <@singularity> identify **********
15:16 <@root> Password accepted, settings and accounts loaded
15:16 <@root> Trying to get all accounts connected...
15:16 <@root> facebook - Logging in: Authenticating
15:16 -!- Irssi: Join to &bitlbee was synced in 8 secs
15:16 <@root> facebook - Logging in: Fetching contacts
15:16 <@root> facebook - Logging in: Connecting
15:16 -!- FB-USER_2 [*********************@facebook] has joined &bitlbee
15:16 -!- ServerMode/&bitlbee [+v FB-USER_2] by localhost
15:16 -!- FB-USER_1 [*********************@facebook] has joined &bitlbee
15:16 -!- ServerMode/&bitlbee [+v FB-USER_1] by localhost
15:16 <@root> facebook - Logging in: Logged in
15:16 <@singularity> fbchats facebook
15:16 <@root> facebook - No chats to display.
15:19 -!- FB-USER_1 [**********************@facebook] has quit [Leaving...]
15:19 <@singularity> fbcreate facebook FB-USER_1 FB-USER_2
15:19 <@root> facebook - Error: (#2) Service temporarily unavailable
15:19 <@root> facebook - Signing off..
```

**Comment by dequis** [comment]
Fixed in db5ae4ba9e65062b9ea3c267a8eb61efc7d890cb (in the development branch)

**Comment by KamalGalrani** [comment]
Thanks :) Works like a charm
